packages/workspace/src/generators/new/generate-preset.ts

```diff
@@ -32,25 +32,33 @@ export function generatePreset(host: Tree, opts: NormalizedSchema) {
       interactive: true,
     },
   });
-  const spawnOptions: SpawnOptions = {
+
+  const newWorkspaceRoot = join(host.root, opts.directory);
+  const spawnOptions: ForkOptions = {
     stdio: 'inherit',
-    shell: true,
-    cwd: join(host.root, opts.directory),
-    windowsHide: false,
+    cwd: newWorkspaceRoot,
   };
   const pmc = getPackageManagerCommand();
-  const executable = `${pmc.exec} nx`;
+  const nxBinForNewWorkspaceRoot = require.resolve('nx/bin/nx', {
+    paths: [join(newWorkspaceRoot, 'node_modules')],
+  });
```

Comment:
There should be another method which handles the `.nx` installation as well where this won't work.